recipes-apps/ovmenu-ng/files/openvario-57-lvds/ovmenu-ng.sh

```diff
@@ -171,6 +173,7 @@ function submenu_xcsoar_lang() {
 		 pl_PL.UTF-8 "Poland" \
 		 cs_CZ.UTF-8 "Czech" \
 	 	 sk_SK.UTF-8 "Slowak" \
+	 	 es_ES.UTF-8 "Espanol" \
```

Comment:
Let's make the additional language a separate PR. I think it can be included independently from the IGC download stuff.

Note, that right now there are 4 copies of this menu scripts for all the variants of openvario (for whatever reason). You probably want to change all of them.

Reply:
OK, no problem with that. I'll remove it.